node-install.sh

```diff
@@ -16,9 +69,17 @@ if [[ $EUID != 0 ]]
 fi
 
 ARCH=$(uname -m)
-echo "Searching latest stable version for $ARCH ..."
-URL='https://nodejs.org/dist/'
-if [[ $ARCH == 'aarch64' ]]
+
+if [[ ! -z "$unofficial_build_version" ]]; then
```

Comment:
`[[ -n $unofficial_build_version ]]` or simply `[[ $unofficial_build_version ]]`. No need for doubled negation, and the quotation here as no effect, same like with left sided strings 🙂.

Reply:
Fixed.